packages/api/src/task/scheduler.ts

```diff
@@ -264,6 +264,12 @@ export class TaskScheduler {
       const asset = await db.asset.get(task.outputAssetId);
       if (asset && asset.source.type == "recording" && asset.source.sessionId) {
         const session = await db.session.get(asset.source.sessionId);
+        const externalSession = await toExternalSession(
+          this.config,
+          session,
+          null,
+          true
+        );
         if (session) {
```

Comment:
Should this `toExternalSession` go to inside this `if`? Since the `session` object can apparently be null (don't know how, but the check may be there for a reason)